public_html/wp-content/plugins/wordcamp-reports/views/html/wordcamp-counts.php

```diff
@@ -135,9 +212,14 @@
 				</td>
 
 				<td class="number">
-					<span class="total"><?php echo number_format_i18n( $event['totals']['organizer'] ); ?></span>
+					<span class="total">Total: <?php echo number_format_i18n( $event['totals']['organizer'] ); ?></span>
```

Comment:
Does this need its own `<td>`, so that it doesn't get displayed in the same column as gender?

Reply:
I changed the layout a bit, is this something you had in mind?
| Before | After |
| ------ | ------|
| <img width="2382" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/18050944/ef252937-953f-41b9-884a-94874b109608"> | <img width="2556" alt="image" src="https://github.com/WordPress/wordcamp.org/assets/18050944/772f9aed-174f-4d49-86e8-78212115cd69"> |